docker/llm/finetune/lora/docker/bigdl-lora-finetuing-entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -x
+source /opt/intel/oneapi/setvars.sh
+export CCL_WORKER_COUNT=$WORLD_SIZE
+export CCL_WORKER_AFFINITY=auto
+
+if [ "$WORKER_ROLE" = "launcher" ]
+then
+  sed "s/:1/ /g" /etc/mpi/hostfile > /home/mpiuser/hostfile
+  export DATA_PATH="/ppml/data/$DATA_SUB_PATH"
+  export MODEL_SAVE_PATH="/ppml/output"
+  sleep 10
+  mpirun \
+    -n $WORLD_SIZE \
+    -ppn 1 \
```

Comment:
-ppn can only be 1 or it can be more than 1?

Reply:
-ppn can only be 1.
This is because we designed that only one process in each pod, and we implemented in a way that processes read their ranks from ENV in pod, which allows only one worker in each pod.